config/env.development

```diff
@@ -6,6 +6,10 @@
 
 COMPOSE_PROJECT_NAME=telescope_api
 
+# Use buildx in docker-compose
```

Comment:
Maybe include a link to what buildx is here?

I'm just curious, do you talk about this your CCP55 teachings?

Reply:
Sure, I'll add one. `buildx` adds a bunch of goodies to `build` (e.g., skipping layers that aren't being used, allow cross-compiling, etc.  It's the default on some systems to use it, and on some it isn't.